lib/charms/lego_base_k8s/v0/lego_client.py

```diff
@@ -278,6 +278,18 @@ def _generate_signed_certificate(self, csr: str, relation_id: int):
             relation_id=relation_id,
         )
 
+    def _get_certificate_fulfillment_status(self) -> str:
+        """Return the status message reflecting how many certificate requests are still pending."""
+        outstanding_requests_num = len(
+            self.tls_certificates.get_outstanding_certificate_requests()
+        )
+        if outstanding_requests_num > 0:
+            return (
+                f"{outstanding_requests_num} pending certificate request, "
+                "check juju debug-log for more information"
+            )
+        return "All certificate requests are fulfilled"
```

Comment:
I think having a ratio would be more useful to get an idea of what have (ex. `3/4 certificate requests fulfilled`)

Reply:
I tried to avoid that as in an ideal case the fulfilled csrs should be removed from the relation data. This is not the case in V3.
I can add it. What do you think?

Reply:
Changed for now, but we can easily go back if you agree with me

Reply:
True. The change is made.

Reply:
Ok good, the pr title should probably change to something like `add ratio of fulfilled certificates in status message` then (as this will be the commit message)

Reply:
Almost there (`ration` instead of `ratio`)!